scripts/update-openshift-versions.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# Script to automatically update OpenShift version annotations based on Red Hat API
+# This script finds the minimum OpenShift version with Extended Update Support Add-On available
+# and updates both bundle.Dockerfile and bundle/metadata/annotations.yaml
+
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(dirname "$SCRIPT_DIR")"
+
+# Red Hat API endpoint for OpenShift lifecycle
+API_URL="https://access.redhat.com/product-life-cycles/api/v1/products?name=Openshift%20Container%20Platform%204"
```

Comment:
what's the source of this URL? why platform 4 rather than just platform? 🤔

Reply:
source: https://docs.redhat.com/en/documentation/red_hat_product_life_cycle_data_api/1.0/html-single/red_hat_product_life_cycle_data_api/index

4 means Openshift Container Platform 4